src/Mapbender/CoreBundle/Resources/public/mapbender.element.legend.js

```diff
@@ -351,7 +361,7 @@
                 if(layers[layidx].children.length > sublayidx) {
```

Comment:
Not directly connected to this issue, but in this class is no variable `layidx` or `sublayidx`! In addition they have very bad names!!!